backend/internal/api/repo.go

```diff
@@ -151,8 +152,8 @@ func Graph(w http.ResponseWriter, r *http.Request, _ *db.UserDetail) {
 		util.WriteErrorf(w, http.StatusBadRequest, GenericErrorMessage)
 		return
 	}
```

Comment:
Why is the err check and return removed? The possibility is low but not 0 that the function returns an error

Reply:
seems like something I forgot... added it back in